tests/functional/spec/thank_you.spec.js

```diff
@@ -52,20 +52,19 @@ describe("Thank You Default View Response Enabled", () => {
       expect(browser.getUrl()).to.contain(ThankYouPage.pageName);
     });
 
-    it("When I navigate to the thank you page, and I have submitted less than 45 minutes ago, Then I should see the option to view my answers", () => {
+    it("When I navigate to the thank you page, and I have submitted less than 40 seconds ago, Then I should see the option to view my answers and countdown timer before it expires", () => {
```

Comment:
Do you need the "before it expires"? It's a set time in your when and you wouldn't put after in the other test. Also might need a "the" in before countdown